src/Microsoft.Health.Dicom.Core/Models/Export/AzureBlobExportOptions.cs

```diff
@@ -6,6 +6,7 @@
 using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
+using System.Text.RegularExpressions;
 using Newtonsoft.Json;
 
 namespace Microsoft.Health.Dicom.Core.Models.Export;
```

Comment:
I need to keep the options in the core lib, as I will eventually migrate it to an "abstractions" package. At the end of the day, we'll need to expose some sort of schema to users of the client for the azure blob export API. However, I have migrated the logic to the blob project, as it makes more sense to centralize it (especially because it makes assumptions about how blob storage works)